web-services/pom.xml

```diff
@@ -40,7 +40,7 @@
         <version.abdera>1.1.3</version.abdera>
         <version.asm>4.0</version.asm>
         <version.axiom>1.2.14</version.axiom>
-        <version.bcprov>1.46</version.bcprov>
+        <version.bcprov>1.67</version.bcprov>
```

Comment:
Can you provide a quick explanation for this version bump?

Reply:
Actually this bump messed up some integration, so reworking